mate-session/gsm-logout-dialog.c

```diff
@@ -443,17 +443,17 @@ gsm_get_dialog (GsmDialogLogoutType type,
                 logout_dialog->default_response = GSM_LOGOUT_RESPONSE_LOGOUT;
 
                 if (gsm_logout_supports_switch_user (logout_dialog)) {
-                        gtk_dialog_add_button (GTK_DIALOG (logout_dialog),
-                                               _("_Switch User"),
+                        gsm_util_dialog_add_button (GTK_DIALOG (logout_dialog),
+                                               _("_Switch User"), "system-users",
                                                GSM_LOGOUT_RESPONSE_SWITCH_USER);
                 }
 
                 gsm_util_dialog_add_button (GTK_DIALOG (logout_dialog),
-                                            _("_Cancel"), "process-stop",
+                                            _("_Cancel"), "gtk-cancel",
```

Comment:
`gtk-cancel` is deprecated, `process-stop` is fine

Reply:
The button label is cancel, i have seen a Mate application with the "correct" icon (red circle with white cross on it).
Sure its name is not "gtk-cancel" but another one. Updated soon.
Thanks.

Reply:
`gtk-cancel` and `process-stop` are the same icon, but `gtk-cancel` is deprecated, so... is better `process-stop`

Reply:
I have different icons between process-stop and the icon used in Caja for example:
![Capture du 2020-06-24 07-31-40](https://user-images.githubusercontent.com/50292710/85504755-ef4e0b00-b5ec-11ea-9746-bdfabb5e79ee.png)
mate-screenshot uses the same than Caja, this is the icon i purpose to use.

Reply:
I have these icons:
![Capture du 2020-06-24 07-39-52](https://user-images.githubusercontent.com/50292710/85505172-07725a00-b5ee-11ea-9916-d5e8a1fb93a4.png)
Without the PR applied, the icon used in the shutdown dialog is one of the first line, all others applications use one of the second. I have read the correct name in the original code. I must investigate if the icon theme is correctly loaded in some applications and not here.

Did you have the same issue, icons different between Caja or Mate-screenshot and here?

Thanks.

Reply:
Using tk3-icon-browser, the icon "process-stop" is the good one, one of the second line.

Reply:
caja and mate-screenshot have the same icon here, process-stop (the same as gtk-cancel):

![2020-06-24_12-33](https://user-images.githubusercontent.com/7734191/85540308-1d484500-b617-11ea-85db-dff9baa64cc9.png)

the icon with X seems to be the icon name `"edit-delete"`

just keep the lines `process-stop` with no changes

Reply:
Could you give me the name of the icon set you use, please?
Thanks.

Reply:
mate-icon-theme

Reply:
Lol, I find where the bug was: my brain! I was perhaps switch to another icon set for testing and i was sure the "process-stop" was different, sorry for the waste of time...
Done, switch back to "process-stop".